user/tests/wiring/serial_loopback/serial_loopback.spec.js

```diff
@@ -0,0 +1,6 @@
+suite('Serial loopback');
+platform('gen3');
```

Comment:
@XuGuohui With recent changes to test runner/device-constants this should be 'gen3', 'gen4', see https://github.com/particle-iot/device-os/pull/2814